sns/parity_math.c

```diff
@@ -1482,10 +1556,14 @@ M0_INTERNAL int m0_sns_ir_failure_register(struct m0_bufvec *recov_addr,
 	block->sib_status = M0_SI_BLOCK_FAILED;
 
 	M0_CNT_DEC(ir->si_alive_nr);
+#if ISAL_ENCODE_ENABLED
+	/* Store failed index in buffer */
```

Comment:
s/Store failed index in buffer/Store failed index in buffer and increase the failure number.

Reply:
Done.